sd-finetuner-workflow/sd-finetune-workflow.yaml

```diff
@@ -0,0 +1,381 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
+  generateName: "sd-finetune-"
+spec:
+  entrypoint: main
+  arguments:
+    parameters:
+    # run_name should be unique on a per-run basis, especially if reporting
+    # to wandb or sharing PVCs between runs.
+    - name: run_name
+    - name: pvc
+      value: 'sd-finetune-data'
+    # Training parameters. Model IDs are hugging face IDs to pull down, or
+    # a path to your model relative to the PVC root.
+    - name: model
+      value: 'CompVis/stable-diffusion-v1-4'
```

Comment:
Are we using CompVis or are we using diffusers?

Reply:
Diffusers is being used, I'll specify that in the comment